tests/bwameth_skip_deduplication.nf.test.snap

```diff
@@ -384,12 +384,15 @@
                 "genome_results.txt:md5,ed81b2fd10b2c548610d546b4e4d3b76",
                 "genome_results.txt:md5,92485f8ac205d7fa1140cdc293f07560",
                 "genome_results.txt:md5,478f711b8ef22e094e4d0ff7fa0dfda0"
+            ],
+            [
```

Comment:
no bams here as well

Reply:
yes these both params skip the bams.......this might be a good way to ensure no bams are produced. so, will go with this as is for now